Description: Makefile: use Debian compiler flags
 This patch simplifies upstream's Makefile to be able to honour Debian's
 compiler flags (e.g. for hardening).
Author: Nicolas Schier <nicolas@hjem.rpa.no>
Last-Update: 2014-10-26

--- sleepenh-1.3.orig/Makefile
+++ sleepenh-1.3/Makefile
@@ -1,4 +1,5 @@
-# just a simple makefile
+CC = $(CROSS_COMPILE)gcc
+CFLAGS += -Wall -Wextra
 
 all: sleepenh manpage
 
@@ -7,15 +8,12 @@ clean:
 
 distclean: clean
 
-sleepenh:
-	gcc -o sleepenh -Wall -O2 sleepenh.c
-
 manpage:
 	if [ ! -e "sleepenh.1.gz" ]; then \
 	cat sleepenh.1 | gzip -9 > sleepenh.1.gz ; fi
 
 install: sleepenh manpage
 	mkdir -p ${DESTDIR}/usr/bin/
 	install -m 0755 -o root -g root sleepenh ${DESTDIR}/usr/bin/sleepenh
 	mkdir -p ${DESTDIR}/usr/share/man/man1/
 	install -m 0644 -o root -g root sleepenh.1.gz ${DESTDIR}/usr/share/man/man1/sleepenh.1.gz
